lib/schema_generator/publisher_content_schema_generator.rb

```diff
@@ -23,7 +23,8 @@ def required
       required_properties = base_content_item["required"] + %w[document_type schema_name]
 
       # TODO: `contact` is a content item that is can be base path less.
```

Comment:
Minor comment: we should update this comment to be consistent with the code (also, it is currently worded incorrectly so that can also be fixed!)